lib/create-response-handler.ts

```diff
@@ -120,3 +177,36 @@ export function createResponseHandler (responseScenarios: ResponseScenarios) {
     }
   }
 }
+
+/**
+ * To make handling the passport.authenticate() callback easier we provide a
+ * createIdentityResponseHandler function. This takes separate callbacks for each of the
+ * response scenarios you have to handle and returns a function that will call the
+ * appropriate callback when called by passport.
+ *
+ * NOTE: Should be used by all new services (not using MSA)
+ *
+ * @param responseScenarios Callbacks to handle each type of response that Verify can return
+ */
```

Comment:
Again, please review this comment carefully (and compare it to its createResponseHandler() counterpart further up in the file).  I don't know how clear this will all be to any third-party developers.  It's a shame we have to have two versions of everything but I can't think of a better way to continue support for the legacy situation.